modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss

```diff
@@ -334,6 +334,12 @@
 		}
 	}
 
+	&.active {
+		.select-field-trigger {
```

Comment:
since this is the only child selector in the group, can we put it on the same line as its parent?